client/app/bundles/HelloWorld/reducers/helloWorldReducer.jsx

```diff
@@ -0,0 +1,16 @@
+import actionTypes from '../constants/helloWorldConstants';
+
+const helloWorldReducer = (state = { name: '' }, action) => {
+  const { type, text } = action;
```

Comment:
shouldn't we destructure inside the case block so it's clear which variables go with which actions?